exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/cloud/cloud_store_config.ipynb

```diff
@@ -33,18 +39,21 @@
    "execution_count": null,
    "id": "a86f5b8661854f35",
    "metadata": {
-    "collapsed": false
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
    },
    "outputs": [],
    "source": [
     "from exasol.nb_connector import github, bfs_utils, cloud_storage\n",
     "from exasol.nb_connector.connections import open_bucketfs_connection, open_pyexasol_connection\n",
-    "from exasol.bucketfs.path import BucketPath\n",
+    "import exasol.bucketfs as bfs\n",
     "import pathlib\n",
     "\n",
     "jar_local_path = github.retrieve_jar(github.Project.CLOUD_STORAGE_EXTENSION, use_local_cache=True)\n",
     "local_name = jar_local_path.name\n",
-    "bfs_path = BucketPath(open_bucketfs_connection(ai_lab_config)) / local_name\n",
+    "bfs_path = bfs.path.BucketPath(local_name, open_bucketfs_connection(ai_lab_config))\n",
```

Comment:
The following should work, too, and would avoid the additional import: 
```python
bucket = open_bucketfs_connection(ai_lab_config) / local_name
```

You can also use `bucket.udf_path`

Reply:
OK, my bad, I mixed up `PathLike` and `BucketLike`.